modules/Registration/components/teacher/TeacherRegistrationForm.helpers.ts

```diff
@@ -0,0 +1,70 @@
+import { zodResolver } from "@hookform/resolvers/zod";
+import { z } from "zod";
+
+export const TeacherFormSchema = z
+  .object({
+    firstName: z
+      .string()
+      .max(255, { message: "First name must be at most 255 characters" })
+      .min(1, "First name is required"),
```

Comment:
This might cause complication down the line. Let's do min first and then max. Same comment for occurrences like this.